### PR TITLE
Add support for getusppe regional affiliates in json.

### PR DESCRIPTION
**Pull request by awong-dev**
Slightly hacky on the functions side, but also not sure it's worth refactoring...

### CODE REVIEW

File: functions/index.js [hunk 1]
```diff
@@ -497,87 +490,25 @@ function toDataByLocation(data) {
...
   const country = get_country_from_path(req);
   if (country === 'makers') {
     loadMakerData(admin, req, res);
     return;
   }
 
-  if (!(country in SHEETS)) {
-    res.status(400).send(`invalid country: ${country} for ${req.path}`);
-    return;
+  let data = null;
```
Comment (nburt) [inline functions/index.js:500]: should data = 'data' by default?

**Comment by nburt** [comment]
I'm seeing that the contact name is in the json file, do we want that or should we be filtering out for privacy?

**Comment by nburt** [comment]
I'm seeing that the contact name is in the json file, do we want that or should we be filtering out for privacy?

File: functions/index.js (hunk 1 above)
Comment (awong-dev) [reply functions/index.js:500]: I think I prefer it as it is?  Can change later if it becomes a problem.

**Comment by awong-dev** [comment]
@nburt The names are the public contact names so they should be displayed.

Thanks for wiring in the override!  Gonna merge for now so functionality is deployed.

**Pull request merged**